examples/miscellaneous/miscellaneous_ex2/miscellaneous_ex2.C

```diff
@@ -113,9 +113,9 @@ int main (int argc, char ** argv)
   // This example requires at least 2D support.
   libmesh_example_requires(2 <= LIBMESH_DIM, "2D support");
 
-  // Check for proper usage.
-  if (argc < 3)
-    libmesh_error_msg("Usage: " << argv[0] << " -f [frequency]");
+  // Check for proper usage. frequency can have 1 or three terms:
```

Comment:
If you're forcing argc >= 4 then frequency must always have 2 terms, no?

Reply:
yes; I think this is okay here because the parameter will be of type number anyway.
But you are right; the comment  is misleading.